cmd/kubelet/app/options/options.go

```diff
@@ -420,6 +420,8 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 
 	fs.Int32Var(&c.RegistryPullQPS, "registry-qps", c.RegistryPullQPS, "If > 0, limit registry pull QPS to this value.  If 0, unlimited.")
 	fs.Int32Var(&c.RegistryBurst, "registry-burst", c.RegistryBurst, "Maximum size of a bursty pulls, temporarily allows pulls to burst to this number, while still not exceeding registry-qps. Only used if --registry-qps > 0")
+	fs.StringVar(&c.ImagePullCredentialsVerificationPolicy, "image-pull-credentials-verification-policy", c.ImagePullCredentialsVerificationPolicy, "") // TODO:
+	fs.StringSliceVar(&c.PreloadedImagesVerificationAllowlist, "preloaded-images-verification-allowlist", c.PreloadedImagesVerificationAllowlist, "List of preloaded images that can be accessed without credential verification. Can only be used with --image-pull-credentials-verification-policy=NeverVerifyAllowlistedImages.")
```

Comment:
no need to expose these as flags, new stuff can be set only in the kubelet config file